### DISCUSSION TITLE
CSS class comming from outside repository

### DISCUSSION THREAD

**Discussion by weber-s** [tailpress/tailpress#211] [Q&A]
Hi!

I am wandering if (and how) tailpress could know I use some classes outside the theme. For instance, if I use contact-form plugin and add some classes in its template, tailpress does not see any change and is not compiling the new classes into my final CSS.

A workaround is to add a html file in my theme, with the same html markup, and so it works... but it is kind of hacky :P 

Is there any possibility to watch also this "in-DB" change?

**Comment by jeffreyvr** [comment]
Hi,

I don't think there is an easy way to watch for classes outside of your theme. 

You could use the `safelist.txt` that is already present in the theme to list the classes you want to include: 
https://github.com/jeffreyvr/tailpress/blob/master/safelist.txt

More complicated and probably time consuming approach would be to write a custom script that grabs all the external classes and automatically puts them in the safelist.txt file.
